go/integration_test/processor/create_test.go

```diff
@@ -549,6 +549,97 @@ func TestProcessor_CreateExistingAccountFails(t *testing.T) {
 	}
 }
 
+func TestProcessor_CodeStartingWith0xEFCanNotBeCreated(t *testing.T) {
+	tests := map[string]struct {
+		revision         tosca.Revision
+		firstInstruction byte
+		success          bool
+	}{
+		"preLondon0xEF": {
+			revision:         tosca.R09_Berlin,
+			firstInstruction: byte(0xEF),
+			success:          true,
+		},
+		"london0xEF": {
+			revision:         tosca.R10_London,
+			firstInstruction: byte(0xEF),
+			success:          false,
+		},
+		"postLondon0xEF": {
+			revision:         tosca.R13_Cancun,
+			firstInstruction: byte(0xEF),
+			success:          false,
+		},
+		"preLondonNo0xEF": {
+			revision:         tosca.R09_Berlin,
+			firstInstruction: byte(vm.STOP),
+			success:          true,
+		},
+		"londonNo0xEF": {
+			revision:         tosca.R10_London,
+			firstInstruction: byte(vm.STOP),
+			success:          true,
+		},
+		"postLondonNo0xEF": {
+			revision:         tosca.R13_Cancun,
+			firstInstruction: byte(vm.STOP),
+			success:          true,
+		},
+	}
+
+	for processorName, processor := range getProcessors() {
+		for testName, test := range tests {
+			t.Run(processorName+"/"+testName, func(t *testing.T) {
+				sender := tosca.Address{1}
+				addressToBeCreated := tosca.Address(crypto.CreateAddress(common.Address(sender), 0))
+
+				initCode := []byte{
+					byte(vm.PUSH1), test.firstInstruction,
```

Comment:
Second instruction?

Reply:
The code that is used to create a new account is the output of the execution of the init code. The init code in the test only returns the `firstInstruction`.